core/src/main/java/kafka/server/SharePartitionManager.java

```diff
@@ -694,11 +695,11 @@ public ShareGroupMetrics(Metrics metrics, Time time) {
                 metrics.metricName(
                     SHARE_ACK_RATE,
                     METRICS_GROUP_NAME,
-                    "The total number of offsets acknowledged for share groups per second."),
+                    "The total number of requests acknowledged for share groups per second."),
                 metrics.metricName(
                     SHARE_ACK_COUNT,
                     METRICS_GROUP_NAME,
-                    "The total number of offsets acknowledged for share groups.")));
+                    "The total number of requests acknowledged for share groups.")));
```

Comment:
I'd prefer "The number of acknowledge requests." Don't get into per second, or in the last 30 seconds or whatever. The measurement period for these things can be modified by configurations which make it hard to give totally precise definitions in practice.